app/src/lib/editors/registry.ts

```diff
@@ -0,0 +1,75 @@
+// This module is a stripped back version of winreg:
+// https://github.com/fresc81/node-winreg
+//
+// I was seeing significant overhead when spawning the process to enumerate
+// the keys found by `reg.exe`, and rather than trying to fix and potentially
+// regress other parts I've extracted just the bit that I need to use.
+
+import { spawn } from 'child_process'
+
+type RegistryEntry = {
+  name: string
+  type: string
+  value: string
+}
+
+const regPath = 'C:\\Windows\\System32\\reg.exe'
```

Comment:
Shouldn't hard code the path should use
path.join(process.env.windir, 'system32', 'reg.exe');